main.py

```diff
@@ -44,28 +46,71 @@ def __init__(self):
             exit('Check if the configuration is set right')
 
     def _run(self):
-        for submission in self._get_submissions():
-            parsed_submission = TitleParser.create_parsed_submission_from_submission(submission)
+        while True:
+            for submission in self._get_submissions():
```

Comment:
None of this loop changed functionality wise.  Its just indented under the new infinite loop.